tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -645,7 +649,7 @@ def _execute_test_suite(self, suite: TestSuiteInfo, test_context: AgentTestConte
 
                         test_success: bool = True
 
-                        test_instance = test.test_class(test_context)
+                        test_instance = test.test_class(test_context, test_args)
```

Comment:
Rather than passing test_args as a new argument to the test, each test_arg should be added to the text_context as a new property (or if the property already exists in the context, the just set its value).

Then AgentPublishTest.run_from_command_line should read the new parameters from the command line and add them to the test context as well. That is, it would parse the publishedVersion argument, instead of the test_args argument